data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,470 @@ mission "Unfettered returning home"
+			`	As you head to the bridge, preparing to take off from the planet, you notice a message on the command panel:`
+			`	"You would be wise to use the Anomalocaris as your flagship for this delivery."`
+				accept
+	on enter
+		system
+			attributes "unfettered"
+		conversation
+			action
+				"old reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)"
+				"reputation: Hai (Unfettered)" >?= 0
+			`As you enter the system, you prepare for a fight to get the cargo through, but it is surprisingly quiet. For the first time, entering Unfettered space does not mean getting shot at by missiles from all sides.`
+	on fail
+		"reputation: Hai (Unfettered)" <?= -1000
+		dialog `The Unfettered and their associates are clearly not happy when they somehow find out you didn't deliver the cargo in time, leaving you a brief message on your command panel: "You will not prove yourself to us by failing our requests."`
+	on visit
+		dialog `You have reached <planet>, but you either did not bring the Anomalocaris, or some of the cargo was on another ship.`
+	to complete
+		has "ship model: Anomalocaris"
+	on complete
+		dialog `After landing, a group of Unfettered gather around your ship and begin unloading the medical supplies, though strangely they don't seem to be in any hurry. Shortly after they've finished, you receive a rather brief message from the Unfettered you bought the ship from. "Thank you for helping our brethren, <last>. Please accept these <payment>."`
+		payment 1500000
+		log "Delivered a pressing food shipment to the Unfettered Hai using an Anomalocaris, which seems to have pleased them."
+
+
+
+mission "Hai Legacy 3"
+	landing
+	name "Ancient Claim"
+	description "Beat the Lightning Bugs in combat. You only have one day, and becoming disabled will result in losing the fight."
+	source "Firelode"
+	deadline 2
+	to offer
+		has "Hai Legacy 2: done"
+		has "ship model: Anomalocaris"
+	on offer
+		conversation
+			`As you land, you realize your ship is the center of attention. One Unfettered that looks like an engineer comes to speak to you, warmer than usual compared to your previous experiences with the Unfettered.`
+			`	"Welcome. Although I do not know how you have obtained this ship, I imagine some of us will see it as a sign. It is a relic of our past, a past we are quite proud of. As such, most Unfettered warriors will say that you need to prove your right to use that ship to us. Luckily for you, I don't believe in any of that. What matters to me is that this ship is very durable, and it's filled with valuable ancient technology.`
+			`	"I even see technology from those savage Korath that we fought valiantly to obtain!"`
+				to display
+					has "outfit: Korath Detainer"
+			`	Her tone suddenly changes as she raises her voice for everyone to hear. "I, Mera, challenge you! The winner will earn the right to own this honorable relic and its mighty weapons!"`
```

Comment:
```suggestion
			`	"Welcome. Although I do not know how you have obtained this ship, I imagine some of us will see it as a sign. You see, it is a relic of our past, a past we are quite proud of. As such, most Unfettered warriors will say that you need to prove your right to use it."
			`	After a sigh, her tone then changes. "But I don't believe in any of that. What does matter to me is that ship is filled with valuable ancient technology and very durable.`
			`	Her eyes light up as she focuses more on your ship. "I even see technology from those savage Korath that we fought valiantly to obtain!" `
				to display
					has "outfit: Korath Detainer"
			`	All of a sudden, she raises her voice for everyone to hear. "I, Mera, challenge you! The winner will earn the right to own this honorable relic and its mighty weapons!"`
```

A bit more narrative adjustments to the writing overall with some specific changes. "To us" is unnecessary in that sentence; there's nobody else there speaking/representing the Unfettered or someone else besides the Unfettered themselves. "Luckily for you" doesn't really seem appropriate because she still wants the ship and wants to fight for it, so regardless of Hai history or her desires in the ship, you still have to fight. That said, her tone should change earlier in the dialog since she greeted you warmly, and all of a sudden is talking about why she (really) likes your ship.